test/test_quiz_mode.py

```diff
@@ -67,6 +67,69 @@ def test_passed_quiz_normal_task(self):
         task = json.loads(response.data)
         assert any(task['id'] == non_golden_task.id for non_golden_task in non_golden_tasks)
 
+    @with_context
+    def test_project_quiz_enabled_and_user_quiz_enabled_result_quiz_mode_enabled(self):
+        '''Test that the user can see quiz mode if the user is In Progress for quiz mode and quiz mode is enabled for the project'''
```

Comment:
and user is enabled not only 'in_progress'.

Reply:
Per discussion https://github.com/bloomberg/pybossa/pull/419#discussion_r362923132